MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -237,6 +244,9 @@ public void enableMuxCoreDebug(boolean enable, boolean verbose) {
 
   @SuppressWarnings("unused")
   public void videoChange(CustomerVideoData customerVideoData) {
+    numberOfPauseEventsSent = 0;
```

Comment:
You'll want to reset these in `programChange`, too

Reply:
OK

Reply:
Upldated